modules/platform/nuxeo-apidoc-server/nuxeo-apidoc-core/src/main/java/org/nuxeo/apidoc/introspection/ComponentInfoImpl.java

```diff
@@ -71,8 +74,18 @@
     public ComponentInfoImpl(BundleInfo bundleInfo, String name) {
         bundle = bundleInfo;
         this.name = name;
-        extensionPoints = new HashMap<>();
-        extensions = new ArrayList<>();
+    }
+
+    @JsonCreator
```

Comment:
For the future, instead of having a `private` constructor for json deserialization; you should consider using libray like Google AutoValue (see: https://github.com/google/auto/issues/138#issuecomment-414452935) for not having to maintain a constructor with every attribute.

Reply:
will have a look with https://jira.nuxeo.com/browse/NXP-28838 (which entails more json serializations)